documentation/pom.xml

```diff
@@ -125,108 +177,4 @@
-                                    <resources>
-                                        <resource>
-                                            <directory>${basedir}/src/main/asciidoc/images</directory>
-                                        </resource>
-                                    </resources>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.jboss.maven.plugins</groupId>
-                        <artifactId>maven-jdocbook-plugin</artifactId>
-                        <configuration>
-                            <sourceDirectory>${basedir}/target/asciidoc-docbook</sourceDirectory>
-                            <sourceDocumentName>index.xml</sourceDocumentName>
-                            <masterTranslation>en-US</masterTranslation>
-                            <formats>
-                                <format>
-                                    <formatName>html_single</formatName>
-                                    <stylesheetResource>classpath:/xslt/org/hibernate/jdocbook/xslt/xhtml-single.xsl
-                                    </stylesheetResource>
-                                    <finalName>index.html</finalName>
-                                </format>
-                                <format>
-                                    <formatName>html</formatName>
-                                    <stylesheetResource>classpath:/xslt/org/hibernate/jdocbook/xslt/xhtml.xsl
-                                    </stylesheetResource>
-                                    <finalName>index.html</finalName>
-                                </format>
-                                <format>
-                                    <formatName>pdf</formatName>
-                                    <stylesheetResource>classpath:/xslt/org/hibernate/jdocbook/xslt/pdf.xsl
-                                    </stylesheetResource>
-                                    <finalName>hibernate_validator_reference.pdf</finalName>
-                                </format>
-                            </formats>
-                            <options>
-                                <xincludeSupported>true</xincludeSupported>
-                                <xmlTransformerType>saxon</xmlTransformerType>
-                                <!-- needed for uri-resolvers; can be ommitted if using 'current' uri scheme -->
-                                <!-- could also locate the docbook dependency and inspect its version... -->
-                                <!-- docbookVersion>1.72.0</docbookVersion -->
-                                <localeSeparator>-</localeSeparator>
-                                <injectionDateFormat>yyyy-MM-dd</injectionDateFormat>
-                                <transformerParameters>
```

Comment:
It's easy in asciidoc:
1. Add a docinfo footer file:
  https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-docs/src/main/asciidoc/index-docinfo-footer.html
2. Set these in the attributes element
  https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-docs/pom.xml#L50